conda_gitenv/deploy.py

```diff
@@ -54,6 +54,11 @@ def deploy_tag(repo, tag_name, target):
 
 
 def create_env(pkgs, target, pkg_cache):
+    # Empty package cache so that it will reinitialised.
+    conda.install.package_cache_ = {}
```

Comment:
@lbdreyer Should you cache the `package_cache_` before it's emptied and then reinstate it? e.g.
```python
original_pc = conda.install.package_cache_
conda.install.package_cache_ = {}
...
stuff
...
conda.install.package_cache_ = original_pc